src/torchio/visualization.py

```diff
@@ -39,6 +39,8 @@ def plot_volume(
     xlabels=True,
     percentiles=(0.5, 99.5),
     figsize=None,
+    figtitle=None,
+    figtitle_y=0.75,
```

Comment:
I'm not sure this is needed, and this value certainly doesn't generalize well, e.g. if you try with `FPG`.